src/prototypes/RoomObject.ts

```diff
@@ -16,12 +55,12 @@ Object.defineProperty(RoomObject.prototype, 'targetedBy', { // List of creep nam
 
 RoomObject.prototype.serialize = function(): ProtoRoomObject {
 	const pos: ProtoPos = {
-		x       : this.pos.x,
-		y       : this.pos.y,
-		roomName: this.pos.roomName
+		x: this.pos.x,
```

Comment:
i can reformat this but in general try to avoid commiting formatting-only changes